src/backend/build.gradle.kts

```diff
@@ -25,66 +24,33 @@ kotlin {
 }
 
 tasks {
-    shadowJar {
-        mergeServiceFiles()
-    }
-    named<JavaExec>("run") {
+    runTask {
         this.args = listOfNotNull(
             "--no-auth",
-            project.properties["live.dev.credsFile"]?.let { "--creds=$it"},
-            project.properties["live.dev.widgetPositionsFile"]?.let { "--widget-positions=$it"},
+            project.properties["live.dev.credsFile"]?.let { "--creds=$it" },
+            project.properties["live.dev.widgetPositionsFile"]?.let { "--widget-positions=$it" },
             project.properties["live.dev.contest"]?.let { "--config-directory=$it" },
             project.properties["live.dev.analyticsTemplatesFile"]?.let { "--analytics-template=$it" },
         )
-        this.workingDir(rootDir.resolve("config"))
-    }
-    task<Copy>("release") {
-        from(shadowJar)
-        destinationDir = rootProject.rootDir.resolve("artifacts")
-    }
-    val jsBuildPath = project.buildDir.resolve("js")
-    val schemasBuildPath = project.buildDir.resolve("schemas")
-    val copyJsAdmin = register<Copy>("copyJsAdmin") {
-        from(project(":frontend").tasks["npm_run_buildAdmin"])
-        destinationDir = jsBuildPath.resolve("admin")
-    }
-    val copyJsOverlay = register<Copy>("copyJsOverlay") {
-        from(project(":frontend").tasks["npm_run_buildOverlay"])
-        destinationDir = jsBuildPath.resolve("overlay")
-    }
-    val copySchema = register<Copy>("copySchema") {
-        val genTask = project(":schema-generator").tasks["gen"]
-        from(genTask.dependsOn)
-        destinationDir = schemasBuildPath.resolve("schemas")
-    }
-    register("buildJs") {
-        dependsOn(copyJsAdmin, copyJsOverlay)
-        outputs.dir(jsBuildPath)
-    }
-    register("buildSchema") {
-        dependsOn(copySchema)
-        outputs.dir(schemasBuildPath)
+        this.workingDir = rootDir.resolve("config")
     }
-}
-
-
 
-sourceSets {
-    main {
-        resources {
-            if (project.properties["live.dev.embedFrontend"] == "true") {
-                srcDirs(tasks["buildJs"].outputs)
+    // Not the best way of doing this, but should work out.
+    processResources {
+        into("schemas") {
+            from(project(":schema-generator").tasks.named("gen"))
```

Comment:
This is not the same, that was here. This would cause regerneration of $root/schemas on every backend build, which is quite unexpected by end-users.

Reply:
Oh, I see. Will fix that.

Reply:
Should be fixed. The solution isn't the most elegant, unfortunately.